### ISSUE TITLE
Updating README is necessary regarding runCodeWithFiles

### ISSUE THREAD

**Issue by chullino** [pythonpad/brython-runner#1]
In README.md, it says `runCodeWithFiles` is run with next command.
```
runner.runCodeWithFiles(files['main.py'], files);
```
while the `files` variable is the nested dictionary.
```
const files = {
    'hello.py': {
        'type': 'text',
        'body': 'print("hello world")',
    },
    'data/': {
        'type': 'dir',
        'body': '',
    },
    'data/image.gif': {
        'type': 'base64',
        'body': 'R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7',
    },
    'main.py': {
        'type': 'text',
        'body': 'import hello\nf = open("data/image.gif", "rb")\nf.close()',
    }
};
```
However, `runCodeWithFiles(code, files)`'s first argument must be code, not the dictionary object.
So, `runner.runCodeWithFiles(files['main.py'], files);` must be changed as `runner.runCodeWithFiles(files['main.py'].body, files);`

By the way, thanks for your great, inspiring, and awesome work!

**Comment by jmbyun** [comment]
Oops. Sorry for the inconvenience you might have had because of my mistake. I'll quickly fix and update the document!

**Comment by jmbyun** [comment]
Fixed with c33030e!
